src/Client.php

```diff
@@ -65,6 +65,8 @@ public function query() {
     /**
      * Executes the given Query and returns the result
      *
+     * This method should not be called manually.
```

Comment:
If you look at the diff again you'll see it was removed, but I bet there's a way of doing it in phpdoc too. But as I stated above, I don't think we should change things around concerning the documentation before the clients are stable.